include/tvm/arithmetic.h

```diff
@@ -210,6 +210,14 @@ class ModularSetAnalyzer {
   Impl* impl_;
 };
 
+/*!
+ * \brief Take GCD of a and b.
+ * \param a The first operand.
+ * \param b The second operand.
+ * \return The result.
+ */
+int64_t ZeroAwareGCD(int64_t a, int64_t b);
```

Comment:
This is an internal function, consider moving this function to an internal header. e.g. const_fold.h(or create a new header)